docs/docs_project/pyproject.toml

```diff
@@ -26,4 +26,5 @@ Django = "^5.1"
 heroicons = "2.6.0"
 gunicorn = "20.1.0"
 asgiref = ">=3.6.0,<4"
-whitenoise = "6.2.0"
\ No newline at end of file
+whitenoise = "6.2.0"
+django-cotton = "^1.3.0"
```

Comment:
@daniboygg I would probably keep it unpinned as we will always want to develop the docs with the latest cotton?